fix i18n extraction

msaqib52 opened the pull request:
@xitij2000 please schedule this PR for review.

aayushagrawal9 commented:
@msaqib52 Can you please provide context (Relevant JIRA ticket/problem description etc) for review

msaqib52 commented:
@aayushagra sorry i forgot to add description. 

In this PR i have fixed following issues related to translation string extraction and translation domain file generation:

1. Added `translations` directory to ignore list because it was causing issues with extraction and should be in ignore list.
2. Updated `extract_translations` make command to create `textjs` domain for JS translation strings, Previously it was merging both JS and other strings into `text` domain due to which JS translation strings were not working.

aayushagrawal9 commented:
:+1:

- [x] I tested this: textjs is extracted separately whereas previously it was combined
- [x] I read through the code
- [ ] I checked for accessibility issues
- [ ] Includes documentation
- [ ] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.
